update/update.sh

```diff
@@ -2,4 +2,12 @@
 set -euo pipefail
 
 cd /bundle_update
+
+if [ -f "$PRE_BUNDLE_UPDATE" ]; then
```

Comment:
Perhaps we don't need this conditional. If the script doesn't exist, I'm thinking it makes sense that the bundle update should fail.

Reply:
Additionally, as we're using `eval` on [line 8](https://github.com/envato/bundle-update-buildkite-plugin/pull/7/files#diff-badfd8b06784e6c545bd35d65105e7ecR8), it doesn't have to be a script. A straight up command could be provided:

```yml
steps:
  - label: ":bundler: Update"
    plugins:
      - envato/bundle-update#v0.7.0:
          update: true
          pre-bundle-update: "apk add --no-progress build-base"
```

Reply:
Interesting. Yep you're right it should fail if you have spedified a script and it does not exist. I will update that.

In regards to providing commands via the yml.. I'm not a fan of that tbh. 
It feels nicer if a script holds all these commands, easier to locate run, debug, etc.. 
I might remove the `eval` and just do a `.`

Reply:
It does provide options to others who may think differently though:

> A whole extra file to store one command

And it's consistent with the top level Buildkite step `command` attribute.

Reply:
humm ok fair enough.. I'll trust that developers wont write a 300 line shell script as a yaml parameter :D